### ISSUE TITLE
Corla client: warn when no manifests before proceeding through define audit

### ISSUE THREAD

**Issue by cjcarlton** [DemocracyDevelopers/colorado-rla#243]

**Comment by cjcarlton** [comment]
Also, if easy, change Green buttons w/ black text (originally changed to improve color contrast). to have white text with a black border.

**Comment by vteague** [comment]
Leaving in Raire integration with corla milestone, because it's important to reconcile properly with the new estimate sample sizes feature.
